src/protocol/thread/node.ts

```diff
@@ -198,67 +214,35 @@ export class NodeFront {
   }
 
   // Load all data for this node that is needed to select it in the inspector.
+  async getComputedStyle() {
+    if (!this._waiters.computedStyle) {
+      this._waiters.computedStyle = defer();
+      try {
+        const { computedStyle } = await this._pause.sendMessage(client.CSS.getComputedStyle, {
+          node: this._object.objectId,
+        });
+        this._computedStyle = new Map();
+        for (const { name, value } of computedStyle) {
+          this._computedStyle.set(name, value);
+        }
+      } catch (e) {
+        this._computedStyle = null;
+      } finally {
+        this._waiters.computedStyle.resolve(this._computedStyle);
+      }
+    }
+
+    if (this._computedStyle) {
+      return this._computedStyle;
+    }
+  }
+
   async ensureLoaded() {
     if (this._loadWaiter) {
       return this._loadWaiter.promise;
     }
     this._loadWaiter = defer();
-
-    await Promise.all([
-      this._pause.sendMessage(client.CSS.getComputedStyle, { node: this._object.objectId }).then(
-        ({ computedStyle }) => {
-          this._computedStyle = new Map();
-          for (const { name, value } of computedStyle) {
-            this._computedStyle.set(name, value);
-          }
-        },
-        () => {
-          this._computedStyle = null;
-        }
-      ),
-      this._pause.sendMessage(client.CSS.getAppliedRules, { node: this._object.objectId }).then(
-        ({ rules, data }) => {
-          this._rules = uniqBy(rules, (rule: AppliedRule) => `${rule.rule}|${rule.pseudoElement}`);
-          this._pause.addData(data);
-        },
-        () => {
-          this._rules = null;
-        }
-      ),
-      this._pause.sendMessage(client.DOM.getEventListeners, { node: this._object.objectId }).then(
-        ({ listeners, data }) => {
-          this._pause.addData(data);
-          this._listeners = listeners.map(listener => ({
-            ...listener,
-            handler: new ValueFront(this._pause, { object: listener.handler }),
-            node: this._pause.getNodeFront(listener.node),
-          }));
-        },
-        () => {
-          this._listeners = null;
-        }
-      ),
-      this._pause.sendMessage(client.DOM.getBoxModel, { node: this._object.objectId }).then(
-        ({ model }) => {
-          this._quads = model;
-        },
-        () => {
-          this._quads = null;
-        }
-      ),
-      this._pause
-        .sendMessage(client.DOM.getBoundingClientRect, {
-          node: this._object.objectId,
-        })
-        .then(
-          ({ rect }) => {
-            this._bounds = rect;
-          },
-          () => {
-            this._bounds = null;
-          }
-        ),
-    ]);
+    await this.getComputedStyle();
```

Comment:
This method seems kind of strange now, and at least needs a comment saying why having a computed style == the node is loaded.  It seems best to remove this method, can we just do that?